### PR TITLE
Registers more functions missing.

### PR DESCRIPTION
**Pull request by Zangetsu38**
Stub more functions missing.
Implement and stub cell_FreeType2.
Reorder cellRtc.

If you want change anything, request me.

For cell_Freetype2 i think is better register this module is alone, because more clear for log and for module cellfont and have very long list function, 249 register in sprx, i have register only 27 name is know, other name is actually unknow.

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellRtc.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include "stdafx.h"
+﻿#include "stdafx.h"
```
Comment (Nekotekina) [inline rpcs3/Emu/Cell/Modules/cellRtc.cpp]: Why you reorder stuff? It doesn't make much sense.

File: rpcs3/Emu/Cell/Modules/cell_FreeType2.cpp [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+	REG_FUNC(cell_FreeType2, FT_Outline_Embolden);
+	REG_FUNC(cell_FreeType2, FT_Render_Glyph);
+	REG_FUNC(cell_FreeType2, FT_RoundFix);
+	REG_FUNC(cell_FreeType2, FT_Select_Charmap);
+
+	REG_FUNC(cell_FreeType2, FT_Set_Char_Size);
+	REG_FUNC(cell_FreeType2, FT_Set_Transform);
+
+	REG_FUNC(cell_FreeType2, FT_Vector_Transform);
+});
```
Comment (Nekotekina) [inline rpcs3/Emu/Cell/Modules/cell_FreeType2.cpp]: No line break

File: rpcs3/Emu/Cell/Modules/sys_net.cpp [hunk 3]
```diff
@@ -1022,6 +1040,221 @@ namespace sys_net
...
+		return CELL_OK;
+	}
+
+	s32 _sys_net_lib_bnet_control()
+	{
+		UNIMPLEMENTED_FUNC(libnet);
+		return CELL_OK;
+	}
+
+	s32  __sys_net_lib_calloc()
```
Comment (Nekotekina) [inline rpcs3/Emu/Cell/Modules/sys_net.cpp]: Two spaces

File: rpcs3/Emu/Cell/Modules/sys_net.cpp [hunk 4]
```diff
@@ -1022,6 +1040,221 @@ namespace sys_net
...
+		return CELL_OK;
+	}
+
+	s32 _sys_net_lib_thread_create()
+	{
+		UNIMPLEMENTED_FUNC(libnet);
+		return CELL_OK;
+	}
+	
+	s32	_sys_net_lib_thread_exit()
```
Comment (Nekotekina) [inline rpcs3/Emu/Cell/Modules/sys_net.cpp]: Tab in the middle

File: rpcs3/Emu/Cell/Modules/sys_net.cpp [hunk 5]
```diff
@@ -1022,6 +1040,221 @@ namespace sys_net
...
+		return CELL_OK;
+	}
+
+	s32 sys_netset_get_if_id()
+	{
+		UNIMPLEMENTED_FUNC(libnet);
+		return CELL_OK;
+	}
+
+	s32	sys_netset_get_status()
```
Comment (Nekotekina) [inline rpcs3/Emu/Cell/Modules/sys_net.cpp]: Tab

File: rpcs3/Emu/Cell/Modules/cellRtc.cpp (hunk 1 above)
Comment (Zangetsu38) [inline rpcs3/Emu/Cell/Modules/cellRtc.cpp]: I have register this in first time in February, for that, just prefer with this correct order

File: rpcs3/Emu/Cell/Modules/sys_net.cpp (hunk 3 above)
Comment (Zangetsu38) [inline rpcs3/Emu/Cell/Modules/sys_net.cpp]: yes thx

File: rpcs3/Emu/Cell/Modules/cell_FreeType2.cpp (hunk 2 above)
Comment (Zangetsu38) [inline rpcs3/Emu/Cell/Modules/cell_FreeType2.cpp]: thx fixed

**Comment by Zangetsu38** [comment]
Is good now i think

**Pull request merged**